### PR TITLE
Error Tracking monitor muted updates

### PR DESCRIPTION
**Pull request by meghanlo**
### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/monitors/types/error_tracking.md [hunk 1]
```diff
@@ -42,8 +39,8 @@ There are two types of alerting conditions you can configure your Error Tracking
 
 | Alerting&nbsp;condition     | Description    | 
 | ---  | ----------- |
-|Count| Alert on issues with a high number of errors. For example, alert for your service whenever more than 500 occurrences of your error happen. |
-|New Issue| Triggers when an issue occurs for the first time. You have the option to be notified if a regression occurs, and set a threshold to reduce alerting fatigue.|
+|High Impact| Alert on issues with a high number impact on your end users. For example, alert for your service whenever more than 500 users are impacted by this error. |
```
Comment (cswatt) [inline content/en/monitors/types/error_tracking.md:42]:
```suggestion
|High Impact| Alert on issues with a high number of impacted end users. For example, alert for your service whenever more than 500 users are impacted by this error. |
```

File: content/en/monitors/types/error_tracking.md [hunk 2]
```diff
@@ -116,12 +113,19 @@ To display triggering tags in the notification title, click **Include triggering
 
 For more information about the **Configure notifications and automations** section, see [Notifications][5].
 
+
+### Muting Monitors
+Error Tracking monitors leverage [Issue States][2] to ensure that your alerts stay focused on high-priority matters, reducing distractions from non-critical issues.
```
Comment (cswatt) [inline content/en/monitors/types/error_tracking.md:118]:
```suggestion
Error Tracking monitors use [Issue States][2] to ensure that your alerts stay focused on high-priority matters, reducing distractions from non-critical issues. 
```

File: content/en/monitors/types/error_tracking.md [hunk 3]
```diff
@@ -116,12 +113,19 @@ To display triggering tags in the notification title, click **Include triggering
 
 For more information about the **Configure notifications and automations** section, see [Notifications][5].
 
+
+### Muting Monitors
```
Comment (cswatt) [inline content/en/monitors/types/error_tracking.md:117]:
```suggestion
### Muting monitors
```

**Review by cswatt — CHANGES_REQUESTED** [review]

File: content/en/monitors/types/error_tracking.md [hunk 4]
```diff
@@ -28,7 +25,7 @@ further_reading:
 
 Datadog Error Tracking automatically groups all your errors into issues across your web, mobile, and backend applications. Viewing errors grouped into issues helps you prioritize and find the problems that are most impactful, making it easier to minimize service downtimes and reduce user frustration.
 
-With [Real User Monitoring][1], [APM][2], or [Logs][6] enabled for your organization, you can create an Error Tracking monitor to alert you when an issue in your web or mobile application, backend service, or logs starts, when it has a high impact, and when it starts regressing.
+With [Error Tracking][1] enabled for your organization, you can create an Error Tracking monitor to alert you when an issue in your web or mobile application, backend service, or logs is new, when it has a high impact, and when it starts regressing.
```
Comment (Guillaume-Barrier) [inline content/en/monitors/types/error_tracking.md:28]: Would bullet points serve well this purpose?

File: content/en/monitors/types/error_tracking.md [hunk 5]
```diff
@@ -42,8 +39,8 @@ There are two types of alerting conditions you can configure your Error Tracking
 
 | Alerting&nbsp;condition     | Description    | 
 | ---  | ----------- |
-|Count| Alert on issues with a high number of errors. For example, alert for your service whenever more than 500 occurrences of your error happen. |
-|New Issue| Triggers when an issue occurs for the first time. You have the option to be notified if a regression occurs, and set a threshold to reduce alerting fatigue.|
+|High Impact| Alert on issues with a high number of impacted end users. For example, alert for your service whenever more than 500 users are impacted by this error. |
```
Comment (Guillaume-Barrier) [inline content/en/monitors/types/error_tracking.md:42]: Do we want to make this change now? It's not consistent with the current behaviour and the tabs in the "Define the search query" section

Comment (meghanlo) [reply content/en/monitors/types/error_tracking.md:42]: This is supported isn't it? Especially with RUM when we call it out as a suggested query

File: content/en/monitors/types/error_tracking.md (hunk 4 above)
Comment (meghanlo) [reply content/en/monitors/types/error_tracking.md:28]: I was trying to get rid of the notion of tracks here

File: content/en/monitors/types/error_tracking.md (hunk 5 above)
Comment (Guillaume-Barrier) [reply content/en/monitors/types/error_tracking.md:42]: I guess we can call it like that
But still not consistent with the tabs in "Define the search query" section

**Review by neko-dd — APPROVED** [review]

**Review by cswatt — APPROVED** [review]

**Pull request merged**